docs/getting-started-guides/vsphere.md

```diff
@@ -79,7 +79,7 @@ Sample Config:
 
 #### Known issues
 
-* [Volumes are not removed from a VM configuration if the VM is down](https://github.com/kubernetes/kubernetes/issues/33061). The workaround is to manually remove the disk from VM settings before powering it up.
+* [Volumes are not removed from a VM configuration if the VM is down](https://github.com/kubernetes/kubernetes/issues/33061). The workaround is to manually remove the disk from VM settings before powering it up. - This issue is fixed in 1.5.3
```

Comment:
Both known issues are fixed in 1.5.3. Right? I would suggest to simply remove the section.

And add "Recommended Kubernetes Version: 1.5.3" under "vSphere Cloud Provider" section.